src/bijectors/leaky_relu.jl

```diff
@@ -0,0 +1,95 @@
+"""
+    LeakyReLU{T, N}(α::T) <: Bijector{N}
+
+Defines the invertible mapping
+
+    x ↦ x if x ≥ 0 else αx
+
+where α > 0.
+"""
+struct LeakyReLU{T, N} <: Bijector{N}
+    α::T
+end
+
+LeakyReLU(α::T; dim::Val{N} = Val(0)) where {T<:Real, N} = LeakyReLU{T, N}(α)
+LeakyReLU(α::T; dim::Val{N} = Val(D)) where {D, T<:AbstractArray{<:Real, D}, N} = LeakyReLU{T, N}(α)
+
+up1(b::LeakyReLU{T, N}) where {T, N} = LeakyReLU{T, N + 1}(b.α)
+
+# (N=0) Univariate case
+function (b::LeakyReLU{<:Any, 0})(x::Real)
+    mask = x < zero(x)
+    return mask * b.α * x + (1 - mask) * x
+end
+(b::LeakyReLU{<:Any, 0})(x::AbstractVector{<:Real}) = b.(x)
+
+function (ib::Inverse{<:LeakyReLU, 0})(y::Real)
+    mask = y < zero(y)
+    return mask * (x / b.α) + (1 - mask) * x
```

Comment:
```suggestion
    return mask * (x / ib.orig.α) + (1 - mask) * x
```

Reply:
Ah, thanks! But how was this not caught by the tests? o.O I'll check.

Reply:
It was, by the tests that run on Travis.